crest/Assets/Crest/Crest/Scripts/Underwater/UnderwaterRenderer.Mask.cs

```diff
@@ -185,6 +265,48 @@ void OnPreRenderOceanMask()
             FixMaskArtefacts(_oceanMaskCommandBuffer, descriptor, _maskTarget);
         }
 
+        void PopulateBoundary(CommandBuffer buffer, RenderTargetIdentifier frontTarget, RenderTargetIdentifier backTarget)
```

Comment:
could you add a comment to explain what "Boundary" means

Reply:
ah its the geometry enclosing the volume. i think it would be a bit more idiomatic to call shapes rasterised for this kind of thing as just "volumes". however if this is going to conflict with other things i'm ok with boundary but i think a comment would be useful.

Reply:
Happy to do a rename but will do it throughout to make it consistent. Prefer to do it now rather than later. I'm thinking about renaming "Water Volume Boundary Geometry" to just "Volume Geometry". Might keep the Mode enum the same but could be renamed to something like:

```csharp
public enum Mode
{
    FullScreen,
    Volume2D,
    Volume3D,
    VolumeFilled,
}
```

or 

```csharp
public enum Mode
{
    FullScreen,
    Portal,
    Volume,
    VolumeFilled,
}
```

LMKWYT

Reply:
ok sounds good. i like the latter enum! only i forgot what VolumeFilled is? (prob the name is fine but a comment above each one might be useful)

Reply:
VolumeFilled is a fly-through volume. I guess I should name it VolumeFlyThrough :^)

Reply:
im really sorry but i dont know what a fly through volume is either? im sure it will click in a second :)

Reply:
It's when you can fly through it and still see the effect. Borrowed from fly through clouds. The "Volume" one (third option) will disappear once camera enters it as it doesn't execute the extra passes for the interior or require a stencil buffer.

Reply:
Done. Let me know if you can think of a better name for volume fly through if you think it is confusing.

Reply:
i wont lie - its still bending my mind each time i revisit, but i cant think of better names either. i think a comment explaining what teach one does would be useful if possible? then i htink its good with the current names.

Reply:
Heh... Sure. Done :^)